README.md

```diff
@@ -113,6 +113,22 @@ git clone https://github.com/sysread/bash-gpt
 export PATH="$PATH:/path/to/bash-gpt"
 ```
 
+## With [Nix](https://nixos.org/)
+_Want an easy way to install nix? Checkout the [Determinite Systems Installer](https://github.com/DeterminateSystems/nix-installer)_
+
+1. Check out repository:
+```bash
+git clone https://github.com/sysread/bash-gpt
+```
+2. Add Env Vars to your shell environment
```

Comment:
How is this done in `nix`? Can you make it more specific by adding something like, `Add the required ENV vars to `$HOME/path/to/wherever/nix/stores/your/bashrc`?